src/app/app-routing.module.ts

```diff
@@ -81,6 +82,11 @@ const routes: Routes = [
   //   canActivate: [AuthGuardService],
   //   component: UserRoutesComponent,
   // },
+  {
+    path: 'vicinity',
```

Comment:
[side panel redundancy problem]
To solve the problem you can change this path to routes/vicinity and utilize router-outlet in routes.component.html